core/src/test/java/org/elasticsearch/action/search/TransportMultiSearchActionTests.java

```diff
@@ -63,15 +63,18 @@ public void testBatchExecute() throws Exception {
         when(actionFilters.filters()).thenReturn(new ActionFilter[0]);
         ThreadPool threadPool = new ThreadPool(settings);
         TaskManager taskManager = mock(TaskManager.class);
-        TransportService transportService = new TransportService(Settings.EMPTY, null, null, TransportService.NOOP_TRANSPORT_INTERCEPTOR,
-            boundAddress -> DiscoveryNode.createLocal(settings, boundAddress.publishAddress(), UUIDs.randomBase64UUID()), null) {
+        TransportService transportService = new TransportService(Settings.EMPTY, null, null,
+                TransportService.NOOP_TRANSPORT_INTERCEPTOR,
+                boundAddress -> DiscoveryNode.createLocal(settings, boundAddress.publishAddress(),
+                        UUIDs.randomBase64UUID()), null) {
```

Comment:
Please revert all line-length changes that are changing lines that otherwise were not impacted by the changes in this PR.